exercises/concept/deep-thought/.docs/instructions.md

```diff
@@ -1,10 +1,14 @@
 # Errors
 
-In this exercise, we will be implementing the programming logic in a package `deepthought`,
-which checks whether an input equals to the absolute answer to life, the universe and everything.
+In this exercise, we will be implementing the programming logic in a package `deep_thought`,
+which checks whether an input equals to [the absolute answer to life, the universe, and everything][deep-thought].
 
 We have a constant `answer` which represents the answer to life, and a function `AnswerToLife` which checks the user input
 and returns a boolean and an error type.
 
 A requirement is also to have a general error `ErrCalculation` that we can at the same time use in our `AnswerToLife` function,
 and export to be used in other programs of ours.
+
+TODO: write instructions step by step
+
+[deep-thought]: https://hitchhikers.fandom.com/wiki/Deep_Thought
```

Comment:
Added a link because even though I read all the 5 books myself, I didn't remember that "deep thought" was the name of the computer.